### PR TITLE
[flyteplugins] Make the TF Job Worker Spec Correct

### PR DESCRIPTION
**Pull request by Future-Outlier**
## Describe your changes
As title.

- [ ] I updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] All commits are signed-off.

### CODE REVIEW

**Review by pingsutw — APPROVED** [review]

File: flyteplugins/go/tasks/plugins/k8s/kfoperators/tensorflow/tensorflow.go [hunk 1]
```diff
@@ -122,7 +122,7 @@ func (tensorflowOperatorResourceHandler) BuildResource(ctx context.Context, task
 		workerReplicaSpec := kfTensorflowTaskExtraArgs.GetWorkerReplicas()
 		if workerReplicaSpec != nil {
 			err := common.OverrideContainerSpec(
-				replicaSpecMap[kubeflowv1.MPIJobReplicaTypeWorker].PodSpec,
+				replicaSpecMap[kubeflowv1.TFJobReplicaTypeWorker].PodSpec,
```
Comment (pingsutw) [inline flyteplugins/go/tasks/plugins/k8s/kfoperators/tensorflow/tensorflow.go:125]: could we add a test for it?

Comment (Future-Outlier) [reply flyteplugins/go/tasks/plugins/k8s/kfoperators/tensorflow/tensorflow.go:125]: Yes no problem, I will do it!

**Review by hamersaw — APPROVED** [review]
There's really no need for a test here right?

**Comment by Future-Outlier** [comment]
> There's really no need for a test here right?

I can add it!

**Pull request merged**